docker/models/containers.py

```diff
@@ -737,7 +737,10 @@ def run(self, image, command=None, stdout=True, stderr=False,
 
         exit_status = container.wait()
         if exit_status != 0:
-            out = container.logs(stdout=False, stderr=True)
+            if not kwargs.get('auto_remove'):
+                out = container.logs(stdout=False, stderr=True)
+            else:
+                out = None
```

Comment:
Hi @shin-, maybe put ``out = None`` before the ``if not kwargs.get('auto_remove')`` looks better. What do you think?

Reply:
👍 Good catch